config/env/default.js

```diff
@@ -28,5 +28,5 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: './modules/core/client/img/brand/favicon.ico'
```

Comment:
@mleanos @codydaig So you happy for me to revert those changes back to the original PR?